core/templates/core/login.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'core/base.html' %}
+{% block title %}Logowanie{% endblock %}
+
+{% block head %}
+<script type="module">
+    const canvas = document.getElementById('starback-bg');
+        const starback = new Starback(canvas, {
+            type: 'dot',
+            quantity: 250,
+            height: window.innerHeight,
+            width: window.innerWidth,
+            backgroundColor: [],
+            randomOpacity: true,
+        });
+        function resizeCanvas() {
+            canvas.width = window.innerWidth;
+            canvas.height = window.innerHeight;
+        }
+        window.addEventListener('resize', resizeCanvas);
+</script>
+{% endblock %}
+
+{% block content %}
+<canvas id="starback-bg" class="fixed -z-[1]"></canvas>
+<div class="h-[80vh] flex justify-center flex justify-items-center place-content-center overflow-hidden">
```

Comment:
> overflow-hidden
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/03b69fce-ea44-4024-8a89-85e86220d4a6)

Reply:
You can't scroll on phone right?

Reply:
Its on laptop, and no, you can't, because overflow-hidden cuts off everything what doesn't fit in screen